services/npm/npm-unpacked-size.tester.js

```diff
@@ -0,0 +1,15 @@
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('Latest unpacked size')
+  .get('/express.json')
+  .expectBadge({ label: 'unpacked size', message: /.* kB/ })
```

Comment:
Can we use the `isFileSize` test validator for the valid tests instead of writing a custom regex please

Reply:
Wasn't aware it existed, I've checked other npm services and bundlephobia service for my code. Will do it.

Reply:
Completed. Resolve if okay.